src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -13,16 +13,37 @@ public class JedisCluster extends UnifiedJedis {
    * Default timeout in milliseconds.
    */
   public static final int DEFAULT_TIMEOUT = 2000;
+  /**
+   * Default amount of attemps for connecting.
```

Comment:
'connecting' is confusing as the word is relevant to different steps of cluster communication.
'attempts'/'maxAttempts' is used only for single command executions.